BinaryEncoding.md

```diff
@@ -335,9 +332,9 @@ The `br_table` operator has an immediate operand which is encoded as follows:
 
 | Field | Type | Description |
 | ---- | ---- | ---- |
-| target_count | `uint16` | number of targets in the target_table |
-| target_table | `uint16*` | target entries that indicate an outer block or loop to which to break |
-| default_target | `uint16` | an outer block or loop to which to break in the default case |
+| target_count | `varuint32` | number of targets in the target_table |
+| target_table | `varuint32*` | target entries that indicate an outer block or loop to which to break |
```

Comment:
Maybe these should stay as u16s? It's nice to have the `br_table` data be fixed size for fast and simple parsing. Also, when running an interpreter, we'll want to index into this list dynamically, so having each element be fixed size is convenient.